tests/_tests_dl_callbacks/experiment.py

```diff
@@ -1,13 +1,30 @@
 # flake8: noqa
 from collections import OrderedDict
+import os
 
 from torch.utils.data import Subset
 
+from catalyst import dl, registry
 from catalyst.contrib.datasets import MNIST
 from catalyst.data.cv import Compose, Normalize, ToTensor
 from catalyst.dl import ConfigExperiment
 
 
+@registry.Callback
+class KeyboardInteruptCallback(dl.Callback):
```

Comment:
let's move it to contrib 😂

Reply:
Are you sure?)
This callback is only for testing purposes)